client/app/app.js

```diff
@@ -10,15 +12,20 @@ import ngOvhProxyRequest from '@ovh-ux/ng-ovh-proxy-request';
 import ngOvhUserPref from '@ovh-ux/ng-ovh-user-pref';
 import ngOvhWebUniverseComponents from '@ovh-ux/ng-ovh-web-universe-components';
 import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import ovhManagerNavbar from '@ovh-ux/manager-navbar';
 import uiRouter from '@uirouter/angularjs';
-
+// import ngOvhSidebarMenu from '@ovh-ux/ng-ovh-sidebar-menu';
+import ngOvhOtrs from '@ovh-ux/ng-ovh-otrs';
```

Comment:
Note: ovh-ux/manager#618 targets some core module changes (as breaking change) to prevent importing this unused dependency. 

Hope it helps you.

Reply:
`@ovh-ux/ng-ovh-otrs` is required by `@ovh-ux/manager-navbar`

Reply:
Both `@ovh-ux/manager-core` and `@ovh-ux/manager-navbar` module are defining the base URL ticket.

Is that make sense?

```sh
$ git grep 'OtrsPopupProvider.setBaseUrlTickets'
packages/manager/modules/core/src/otrs/index.js:    OtrsPopupProvider.setBaseUrlTickets(get(REDIRECT_URLS, 'support'));
packages/manager/modules/navbar/src/assistance-menu/index.js:    OtrsPopupProvider.setBaseUrlTickets(ASSISTANCE_URLS[coreConfigProvider.getRegion()].ticket);
```